devel/redo/Portfile

```diff
@@ -1,10 +1,10 @@
 PortSystem              1.0
 
 name                    redo
-version                 0.06
-revision                1
+version                 0.42
```

Comment:
Please, fix the commit message to match the actual `version`

Reply:
Fixed, thanks😅